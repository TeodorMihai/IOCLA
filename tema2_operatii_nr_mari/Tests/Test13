9999FA22 * -9999DFFF
