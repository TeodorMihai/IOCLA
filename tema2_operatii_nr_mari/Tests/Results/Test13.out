22FA9999
FFDF9999
DEC595F1EA59295C
-5C2959EAF195C5DE
